### PR TITLE
 Add slack messaging package

### PR DESCRIPTION
**Pull request by ThatcherK**
ticket #1252

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -28,3 +28,4 @@ wagtail-metadata>=4.0.2,<4.1.0
 whitenoise==5.2.0
 wagtailmedia>=0.11.0,<1.0.0
 PyGithub==1.54.1
+git+https://github.com/devinit/slack_messages.git
```
Comment (wakibi) [inline requirements.txt:31]: Can you investigate possibility of having our own NPM repo where we can publish this, as opposed to pulling it from Github directly?

Comment (wakibi) [reply requirements.txt:31]: Also, can you investigate possibility of using (jinja2) templates to make the text more rich and formatted?

**Review by wakibi — CHANGES_REQUESTED** [review]

File: di_website/settings/base.py [hunk 2]
```diff
@@ -86,6 +86,7 @@
     'wagtailmetadata',
     'django_google_optimize',
     'markdownify.apps.MarkdownifyConfig',
+    # 'django_slack',
```
Comment (edwinmp) [inline di_website/settings/base.py:89]: Remove this entirely.

**Review by edwinmp — CHANGES_REQUESTED** [review]

**Comment by edwinmp** [comment]
@ThatcherK @wakibi is this done? If so, are we good to remove the intentional bug and merge this?

**Comment by ThatcherK** [comment]
No It's not, there's a bug that I haven't yet figured out

**Comment by edwinmp** [comment]
Changing it to draft until you're free to get back to it.

**Pull request closed without merging**